lib/mayaUsd/render/vp2RenderDelegate/CMakeLists.txt

```diff
@@ -18,7 +18,22 @@ target_sources(${PROJECT_NAME}
 )
 
 set(headers
+    basisCurves.h
+    bboxGeom.h
+    CMakeLists.txt
+    debugCodes.h
+    draw_item.h
+    instancer.h
+    material.h
+    mesh.h
     proxyRenderDelegate.h
+    render_delegate.h
+    render_param.h
+    render_pass.h
+    resource_registry.h
+    sampler.h
+    task_commit.h
+    tokens.h
```

Comment:
This list should *only* include `proxyRenderDelegate.h`. All the rest are private headers.

Reply:
Resolved in #391.